src/lib/draw-and-pick.js

```diff
@@ -351,7 +376,13 @@ function getPickedColors(gl, {
             viewport: layer.context.viewport
           }),
           uniforms: Object.assign(
-            {renderPickingBuffer: 1, pickingEnabled: 1},
+            // TODO: Update all layers to use 'picking_uActive' (picking shader module)
+            // and then remove 'renderPickingBuffer' and 'pickingEnabled'.
+            {
+              renderPickingBuffer: 1,
+              pickingEnabled: 1,
+              picking_uActive: 1
```

Comment:
`renderPickingBuffer` and `pickingEnabled` do not belong to picking module, they belong to existing picking functionality in deck.gl. But I can set `picking_uActive` with `updateModuleSettings`, as commented `renderPickingBuffer` and `pickingEnabled` will eventually be removed.